### ISSUE TITLE
npm dist-tag add/rm doesn't work with 2FA

### ISSUE THREAD

**Issue by helfer** [npm/npm#19346]
#### I'm opening this issue because:

  - [x] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?

running `npm dist-tag add package@version tag` fails with a 401 if 2FA is enabled (without prompting for OTP).

#### How can the CLI team reproduce the problem?

Turn on 2FA (which hopefully you already have) and run `npm dist-tag add package@version sometag` for a package that you own. It will fail. Now turn off 2FA and try the same command. It will succeed.

### supporting information:

 - `npm -v` prints: 5.5.1
 - `node -v` prints: v8.9.1
 - `npm config get registry` prints: https://registry.npmjs.org/
 - Windows, OS X/macOS, or Linux?: macOS
 - Network issues:
   - Geographic location where npm was run:
   - [ ] I use a proxy to connect to the npm registry.
   - [ ] I use a proxy to connect to the web.
   - [ ] I use a proxy when downloading Git repos.
   - [ ] I access the npm registry via a VPN
   - [ ] I don't use a proxy, but have limited or unreliable internet access.
 - Container:
   - [ ] I develop using Vagrant on Windows.
   - [ ] I develop using Vagrant on OS X or Linux.
   - [ ] I develop / deploy using Docker.
   - [ ] I deploy to a PaaS (Triton, Heroku).

**Comment by mcollina** [comment]
Confirming the same issue.

**Comment by mcollina** [comment]
The documentation prompts that someone should use `--otp`  to use this command with 2FA. However, the OTP dialog is not printed.

**Comment by mattberther** [comment]
@mcollina I think its `npm dist-tag --otp="123456" ...`. You have to pass the otp code with the --otp option.

**Comment by ethanresnick** [comment]
Yeah, it'd be nice if this prompted for the OTP, like `npm publish` does, rather than requiring the user to know about the otp option.
